lxd/device/config/device_runconfig.go

```diff
@@ -33,6 +33,12 @@ type RootFSEntryItem struct {
 	Opts []string // Describes the mount options associated with the filesystem.
 }
 
+// USBDeviceItem represents a set of USB devices matched from a single LXD USB device specification.
+type USBDeviceItem struct {
+	DeviceName         string
+	USBHostDevicePaths []string
```

Comment:
Two comments here:

1. This doesn't need to contain the word "USB" as it is inherently related to USB being part of the "USBDeviceItem" struct.
2. As `RunConfig.USBDevice` is itself a slice, we don't need to have each individual `USBDeviceItem` support multiple `USBHostDevicePaths`.

So lets make this field just `HostDevicePath string`

Reply:
Wait, that's the part that doesn't work. We either must modify device name here, so it's no longer tracking what LXD was told by the operator OR we need to use a slice of host device paths. I understand you seem to want the former, but what will make the device names unique then?

Reply:
See https://github.com/lxc/lxd/pull/9497#discussion_r745791919